MvvmCross.Forms/Platform/Mac/Platform/MvxFormsMacSetup.cs

```diff
@@ -63,7 +63,7 @@ public MvxFormsApplication FormsApplication
 
         protected abstract MvxFormsApplication CreateFormsApplication();
 
-        protected override IMvxMacViewPresenter CreatePresenter()
+        protected override IMvxMacViewPresenter CreateViewPresenter()
```

Comment:
Do we need this breaking change? `CreateViewPresenter` is much cleaner, but we don't want users to hate us / not to move forward to 6.0 because of tons of breaking changes. Maybe keep both and mark `CreatePresenter` as obsolete?